worker/include/Channel/UnixStreamSocket.hpp

```diff
@@ -73,6 +73,8 @@ namespace Channel
 		void OnConsumerSocketClosed(ConsumerSocket* consumerSocket) override;
 
 	private:
+		size_t SendImpl(const void* nsPayload, size_t nsPayloadLen);
```

Comment:
We never place method declarations within class members. Can you put it above `/* Pure virtual methods inherited from ConsumerSocket::Listener. */` within a new `private:` block?

Reply:
OK, I'll do it, I'm new with this project, some coding style is not familiar with me.